products/centos-stream.md

```diff
@@ -5,7 +5,7 @@ tags: linux-distribution
 iconSlug: centos
 permalink: /centos-stream
 versionCommand: cat /etc/redhat-release
-releasePolicyLink: https://wiki.centos.org/About/Product
+releasePolicyLink: https://wiki.centos.org/About(2f)Product.html
```

Comment:
Perhaps we better use https://centos.org/stream9/ for `releasePolicyLink` and add https://wiki.centos.org/About(2f)Product.html with a remark at the end of the file saying 8 was a transition version for migration to stream. WDYT?

Reply:
I am OK with that.

This link was broken for a few weeks, I was waiting for a new documentation to appear following the archival of the old wiki but did not saw anything coming until now.